ci/test-conformance-eks.sh

```diff
@@ -230,8 +203,9 @@ function run_conformance() {
 
     if [[ "$TEST_FAILURE" == false ]]; then
         echo "=== SUCCESS !!! ==="
+    else
+        echo "=== FAILURE !!! ==="
     fi
-    echo "=== FAILURE !!! ==="
```

Comment:
@Dyanngg looks like we do the same thing in other scripts as well: we always echo "FAILURE" even in case of success :) ?

Reply:
Seems like it :P I think it got missed because in Jenkins everything is running under set +exu, so I must have thought that the "FAILURE" is printed because it was in the code. Will post a fix for this.

Reply:
You may want to update the gke script to remove the `sed`s now that we no longer require the Node MTU and the Service CIDR in the Agent configuration.

Reply:
Opened a PR for this.